### PR TITLE
Feat/csv upload ingestion

### PR DESCRIPTION
**Pull request by onpaws**
Additional changes to better match designer comps; and improve usability of `<Upload />`

### CODE REVIEW

**Comment by Gelio** [comment]
Could you add a before/after screenshot from the gradient change?

This way I could review this easier than having to check out the PR myself and look for it in the UI (~20 minutes) :slightly_smiling_face:

File: tdesign/src/themes/muiTheme.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export const muiTheme = ({
     rightMarginInsideNavProfileMenu: `max(2rem, calc((100vw - ${breakpointValues.desktop}px) / 2) + ${magicNumbers.nav.profile.fromRightOfPageToLeftOfMenu})`,
 
     brandGradient: `linear-gradient(90deg, rgb(249 69 105 / 100%) 0%, rgb(255 128 153 / 50%) 100%)`,
-    pinkBackgroundGradient: `linear-gradient(185.2deg, #FDF2F4 24.37%, rgba(253, 242, 244, 0) 103.81%)`,
+    pinkBackgroundGradient: `linear-gradient(180deg, #FDF2F4 0%, #FDF3F5 100%)`,
```
Comment (Gelio) [inline tdesign/src/themes/muiTheme.ts:73]: The gradient looks pretty solid to me :smile: By solid I mean that I cannot see the color change in any way. Is that expected? Looking at the hex colors, it's almost the same color

image

Did you take it straight from Figma?

Comment (onpaws) [reply tdesign/src/themes/muiTheme.ts:73]: Yes, Kush and I noticed that also, we actually hopped on a brief call about exactly this. 
This gradient is very subtle, almost to the point where you can't see it, but it's there and different from the previous.

There is a Figma component this came from, I'm not sure where it is at the moment but Kush may remember if you're trying to track it down.

(FYI, historically there were also a couple cases where a "gradient" had both colors set to identical values, which is a noop. I think those have been resolved though)

Comment (Gelio) [reply tdesign/src/themes/muiTheme.ts:73]: > There is a Figma component this came from, I'm not sure where it is at the moment but Kush may remember if you're trying to track it down.

Adding it to the PR's description / ClickUp ticket would certainly make it easier to find :slightly_smiling_face: I tracked it down and added it to the ClickUp ticket.

To me, it does not look like we should be using a gradient at all in the CSV preview/ingestion flow. The background behind the public/private header on the CSV preview pager looks like a solid color to me. The fact that it uses a gradient for the background is basically a hack that makes the background solid.

Looking at the definition of the gradient in Figma, I see:

```css
linear-gradient(185.2deg, #FDF2F4 24.37%, rgba(253, 242, 244, 0) 103.81%);
```

which is exactly the previous value. I suggest reverting this commit and not using `pinkBackgroundGradient` at all in the CSV preview.

**Comment by milesrichardson** [comment]
We've already merged the corresponding MR in the parent repo for this branch. 

It looks like there are some changes you want to make here, but it doesn't make sense to address them on this branch, since the changes won't be in the parent repo anyway.

@onpaws Can you capture this in a clickup and do it on a new branch in this repo as part of the follow-up refinements?

**Comment by onpaws** [comment]
Merging and made a CU for the https://github.com/splitgraph/splitgraph.com/pull/151#discussion_r776697580

**Pull request merged**